README.md

```diff
@@ -132,28 +234,109 @@ The Hardware UUID you wish to set (usually generated with `uuidgen`).
       {
         "port_forwarding_guest_port": 8080
       }
-    ]
+    ],
+    "vm_name": "macos-from-packer-base_10.15.7"
   }]
 ```
 
+* `ram_size` (String)
+
+The size in "[0-9]+G" format, defaults to `2G`.
+
+* `registry-path` (String)
+
+The registry URL (will use your default configuration if not set).
+
+* `remote` (String)
+
+The registry name (will use your default configuration if not set).
+
+* `source_vm_tag` (String)
+
+Specify the tag of the VM we want to clone instead of using the default.
+
+* `stop_vm` (Boolean)
+
+Whether or not to stop the vm after it has been created, defaults to false.
+
+* `update_addons` (Boolean)
+
+Update the vm addons. Defaults to false.
+
+## Post Processors
+
+### veertu-anka-registry-push
+
+#### Required Configuration
+
+* `type` (String)
+
+Must be `veertu-anka-registry-push`
+
+#### Optional Configuration
+
+* `cacert` (String)
+
+Path to a CA Root certificate.
+
+* `cert` (String)
+
+Path to your node certificate (if certificate authority is enabled).
+
+* `description` (String)
+
+The description of the tag.
+
+* `insecure` (Boolean)
+
+Skip TLS verification.
+
+* `key` (String)
+
+Path to your node certificate key if the client/node certificate doesn't contain one.
+
+* `local` (Boolean)
+
+Assign a tag to your local template and avoid pushing to the Registry.
+
+* `registry-path` (String)
+
+The registry URL (will use your default configuration if not set).
+
+* `remote` (String)
+
+The registry name (will use your default configuration if not set).
+
+* `remote-vm` (String)
+
+The name of a registry template you want to push the local template onto.
+
+* `tag` (String)
+
+The name of the tag to push (will default as 'latest' if not set).
+
 ## Development
```

Comment:
We should add a note on how to work with `gomock`. Especially how to (re-)create the mocks.